app-modules/interaction/src/Filament/Resources/InteractionDriverResource/Pages/ListInteractionDrivers.php

```diff
@@ -49,10 +59,86 @@
 use Filament\Tables\Table;
 use Illuminate\Database\Eloquent\Builder;
 
-class ListInteractionDrivers extends ListRecords
+class ListInteractionDrivers extends ListRecords implements HasForms
 {
+    use InteractsWithForms;
+
     protected static string $resource = InteractionDriverResource::class;
 
+    /** @var array<string, mixed> */
+    public ?array $data = [];
+
+    protected static string $view = 'interaction::filament.pages.list-interaction-drivers';
+
+    private ?InteractionManagementSettings $settings = null;
+
+    public function mount(): void
+    {
+        if (InteractionMetadataFeature::active()) {
+            $settings = $this->getSettings();
+
+            $this->data = [
+                'is_driver_enabled' => $settings->is_driver_enabled,
+                'is_driver_required' => $settings->is_driver_required,
+            ];
```

Comment:
Just to be safe, you should use `$this->form->fill()` instead of directly setting the data property to ensure all the hooks run